integration/testnet/network.go

```diff
@@ -606,6 +695,39 @@ func (net *FlowNetwork) WriteRootSnapshot(snapshot *inmem.Snapshot) {
 	require.NoError(net.t, err)
 }
 
+func followerNodeInfos(confs []ConsensusFollowerConfig) ([]bootstrap.NodeInfo, error) {
+	var nodeInfos []bootstrap.NodeInfo
+
+	// TODO: remove this, networking keys should be provided by the consensus follower config.
+	// get networking keys for all followers
+	networkKeys, err := unittest.NetworkingKeys(len(confs))
+	if err != nil {
+		return nil, err
+	}
+
+	// get staking keys for all followers
+	stakingKeys, err := unittest.StakingKeys(len(confs))
```

Comment:
why do we need to generate staking keys?

Reply:
Hmm, I think in theory we can remove them. I'm just not sure whether the rest of the code will complain? But you can try it